recipes/djangorestframework-filters/LICENSE

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2017, conda-forge
```

Comment:
We cannot add any project which does not belong to us as licensed as ``conda-forge``. That is a legal document, so, we cannot generate them :/

Reply:
@marcelotrevisani are you saying that this should be packaged with the MIT license, given to https://github.com/allisson/django-rest-framework-role-filters or it should have the above changes to the BSD license file?